src/components/tabBars.js

```diff
@@ -78,26 +78,30 @@ export const TabBar = ({
   }
 
   return div({
-    role: 'navigation',
-    'aria-label': props['aria-label'], // duplicate the menu's label on the navigation element
-    'aria-labelledby': props['aria-labelledby']
+    style: Style.tabBar.container
```

Comment:
Style moved up to parent div because we still want it to appear that the "pill" and the menu are contained in the same horizontal element.